samples/nextjs/next.config.js

```diff
@@ -2,7 +2,7 @@ const withTM = require('next-transpile-modules')(['@sitecore-jss/sitecore-jss-ne
   resolveSymlinks: false,
 });
 const path = require('path');
-const prodConfig = require('./next.config.prod');
+const prodConfig = require('./next.config.base');
```

Comment:
Still using "prod" for variable name and in comments both here and in `jss-create.js`. Rename to match new "base" naming.

Reply:
@ambrauer Added, look again, please